lua/tardis/parts/legacy/legacy_typewriter.lua

```diff
@@ -0,0 +1,22 @@
+--Legacy Interior - Typewriter
+
+local PART = {}
+PART.ID = "legacy_typewriter"
+PART.Name = "Legacy Typewriter"
+PART.Model = "models/drmatt/tardis/typewriter.mdl"
+PART.AutoSetup = true
+PART.Collision = true
+PART.Animate = true
+
+if SERVER then
+    function PART:Use(ply)
+        if not ply:IsPlayer() then return end
+        ply:SendLua([[
```

Comment:
It would be cool to have a proper function to do this within the screen module and just an API for `TARDIS:SwitchScreen('screenname', optionalPly)` on the server which does this on the client essentially

Reply:
Now you have your TARDIS:PopToScreen on the client, just add a server variant which uses the SendMessage feature to make the client do it and job done!